### ISSUE TITLE
Medical Imaging and the Browser

### ISSUE THREAD

**Issue by karchie** [boulder-js/talks#6]
### Abstract

Medical imaging technologies such as X-Ray, CT, MRI, and ultrasound are important in both medicine and in scientific research. I will give an introduction to the imaging information infrastructure of hospitals and radiology practices, focusing on systems like PACS (Picture Archiving and Communication System) and RIS (Radiology Information System) and the DICOM standard that they use to interoperate. I'll then dive into some of the technical challenges as image viewers and reporting applications increasingly move from legacy Java and C++ code into browser-based JS clients with local or cloud backends.

Length: can go as long as an hour, or I can leave bits out and fit it into 25 minutes.
Note: This would not be my first tech talk but I haven't attended much less presented at Boulder JS. Also, while I do bring it around to JS in ways that I think are interesting, that's not mostly what this talk is about. I'm open to presenting in December if that's what allows there to be a meeting but would prefer a little later so I can attend and get a sense for this group.

### Date

19.12.2024

### This will be my first tech talk

- [ ] This will be my first tech talk

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/boulder-js/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by PatrickHeneise** [comment]
Thanks @karchie, that sounds awesome! If 12/18 works for you, we can schedule the talk for that event.

**Comment by karchie** [comment]
> Thanks @karchie, that sounds awesome! If 12/18 works for you, we can schedule the talk for that event.

yes, great, can do then @PatrickHeneise . what's the display tech? might determine which laptop I bring.

**Comment by PatrickHeneise** [comment]
Regular HDMI interface; There's a relatively big (4K?) TV screen in the room.

**Comment by karchie** [comment]
@PatrickHeneise this event is Thu 12/19 not Wed 12/18, correct?

**Comment by PatrickHeneise** [comment]
@karchie correct, the event is tomorrow, Thursday 12/19

**Comment by PatrickHeneise** [comment]
Thanks @karchie for the talk! Feel free to upload the slides here if you can share them.
